source/more-info/unhealthy/oserror_bad_message.markdown

```diff
@@ -0,0 +1,17 @@
+---
+title: "Operating System error: Bad message"
+description: "More information on what to do when OS errors mark the installation as unhealthy."
+---
+
+## The issue
+
+The Supervisor received Operating System errors during file operation (Python OSError exceptions). The error number reported was 74, "Bad message", which is typically raised on file system corruption.
```

Comment:
```suggestion
The Supervisor received Operating System errors during file operation (Python OSError exceptions). The error number reported was 74, "Bad message", which is typically raised on file system corruption. File system corruption can happen due to a power outage or when the system otherwise did not get shut down properly. 
```